### DISCUSSION TITLE
Getting errors when trying to build a project?

### DISCUSSION THREAD

**Discussion by Gamerfiend** [gen2brain/raylib-go#328] [Q&A]
The errors seem to be dependency related. I'm on Windows 11, and have installed the TDM compiler from the README. :)

```
# github.com/gen2brain/raylib-go/raylib
vendor\github.com\gen2brain\raylib-go\raylib\cgo_windows.go:7:10: fatal error: external/glfw/src/context.c: No such file or directory
    7 | #include "external/glfw/src/context.c"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

Compilation finished with exit code 1
```

I checked my `vendor/github.com/gen2brain/raylib-go/raylib` directory and it doesn't include the `external` folder that is on Github.

**Comment by JupiterRider** [comment]
@Gamerfiend This is known "issue": #281 
@gen2brain Btw. Are you alive? I haven't heard from you in a while.
